src/index.js

```diff
@@ -6,5 +6,6 @@ import Tabs from './tabs'
 import Toggle from './toggle'
 import Popover from './popover'
 import Slideover from './slideover'
+import ColorPreview from './color_preview'
```

Comment:
This is loading `src/color_preview.js` but the file name is `src/color_picker.js`

That's probably the reason for the tests failing.

I think I search for "color picker" on Google most times, maybe we just call it that?

```javascript
import ColorPreview from './color_preview'
```

The export will need to get updated too.

Reply:
Ah good catch, I planned to call it `color_preview` since the Stimulus functionality is associated to showing a preview but I'm open to changing the name if `color_picker` if you prefer that.

It looks like the tests are fixed now after force pushing your change.

Although locally I still get the same error if I open the docs `index.html` and try it out even if I comment out that one script that you called out at the top of the file.

Also what's your thoughts on adding the forms plugin to this repo here so that the demo site showcases the component nicely? Should we add a callout in the docs to mention it requires the forms plugin if you plan to use the demo's HTML?